cog/admin_role.py

```diff
@@ -113,11 +121,16 @@ async def fetch_user_by_name(name):
                 color = discord.Color.blurple()
             )
 
+            async def recordDB(btnid, type, count,recipient):
```

Comment:
(From Pylint)

Didn't meet the naming convension.

Could rename to __record_db instead.